doc/source/user_guide/merging.rst

```diff
@@ -724,6 +724,27 @@ either the left or right tables, the values in the joined table will be
           labels=['left', 'right'], vertical=False);
    plt.close('all');
 
+You can merge a mult-indexed Series and a DataFrame, if the names of
+the MultiIndex correspond to the columns from the DataFrame. Transform
+the Series to a DataFrame using :meth:`Series.reset_index` before merging,
+as shown in the following example.
+
+.. ipython:: python
+
+   df = pd.DataFrame({"Let": ["A", "B", "C"], "Num": [1, 2, 3]})
+   df
+
+   ser = pd.Series(
+       ["a", "b", "c", "d", "e", "f"],
+       index=pd.MultiIndex.from_arrays(
+           [["A", "B", "C"] * 2, [1, 2, 3, 4, 5, 6]], names=["Let", "Num"]
+       ),
+   )
+   ser
+
+   result = pd.merge(df, ser.reset_index(), on=['Let', 'Num'])
```

Comment:
just

```
   pd.merge(df, ser.reset_index(), on=['Let', 'Num'])
```

Reply:
ok just fix this in PR  #32370  - looking at the rendered docs - how are the diagrams of the df displayed ?

Reply:
> ok just fixed this in PR #32370

thanks

> how are the diagrams of the df displayed ?

since we have a Series and a DataFrame, that format is probably not applicable.

Reply:
ok - thanks. PR #32370 is all green now.